v2v-helper/migrate/migrate.go

```diff
@@ -545,16 +533,27 @@ func (migobj *Migrate) ConvertVolumes(ctx context.Context, vminfo vm.VMInfo) err
 				return errors.Wrap(err, "failed to run ntfsfix")
 			}
 		}
-		// Turn on DHCP for interfaces in rhel VMs
-		if strings.ToLower(vminfo.OSType) == constants.OSFamilyLinux {
-			if strings.Contains(osRelease, "rhel") {
+
+		if virtv2v.IsRHELFamily(osRelease) {
+			// If RHEL family, we need to inject a script to make interface come up with DHCP,
+			// We preserve the ip because we have a port created with the same IP
+			// If NM is present, we inject a script to force DHCP on first boot.
```

Comment:
please also mention this DHCP is openstack DHCP, which will give same IP as the port